rpc/lib/types/types.go

```diff
@@ -135,10 +137,14 @@ type WSRPCConnection interface {
 	GetRemoteAddr() string
 	WriteRPCResponse(resp RPCResponse)
 	TryWriteRPCResponse(resp RPCResponse) bool
+	GetEventSubscriber() EventSubscriber
+}
 
-	AddSubscription(string, interface{}) error
-	DeleteSubscription(string) (interface{}, bool)
-	DeleteAllSubscriptions()
+// EventSubscriber mirros tendermint/tendermint/types.EventBusSubscriber
```

Comment:
or it's ok to use `tmtypes` here?

Reply:
hmm good question. let's keep the duplication for now and think about how we might improve package structure for things like this